test/models/collapsible_map_test.rb

```diff
@@ -394,6 +394,18 @@ def test_mapping_one_location
     assert_obj_arrays_equal([loc], mapset.underlying_locations)
   end
 
+  # CollapsibleCollection should ignore large locations and observations with
+  # vague locations like "California", when mapping multiple locations.
+  # (:california is currently one of the locations for a "Boletus edulis" obs.)
+  def test_mapping_collections_ignores_vague_locations
+    bol = observations(:boletus_edulis_obs)
+    obss = Observation.where(text_name: bol.text_name).to_a
+    coll = Mappable::CollapsibleCollectionOfObjects.new(obss)
+    coll.mapsets.each do |mapset|
+      assert_not(mapset.underlying_locations.include?(locations(:california)))
```

Comment:
Would this work and do you like it:
```suggestion
assert(mapset.underlying_locations.none?(locations(:california)))
```

Reply:
I generally like anything that teaches me a new syntax, and i was not aware that was equivalent.  But... hmm... i find that syntax a little harder to understand, I reckon.

Reply:
Ah. Rails adds the method `exclude?` to Enumerable and that seems just right. Edited!